client/me/security-u2f-key/index.jsx

```diff
@@ -0,0 +1,150 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
```

Comment:
It's nice to import a specific icon for use like this:

https://github.com/Automattic/wp-calypso/blob/c66660e2ceb5f10fbbde84fd9ff3beb3b5bd1644/client/components/tinymce/plugins/insert-menu/menu-items.jsx#L9-L13

Reply:
Doesn't really matter anymore since we started async loading gridicons.